internal/dev_server/ui/package.json

```diff
@@ -2,19 +2,20 @@
   "name": "ldcli-dev-server-ui",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
   "scripts": {
     "dev": "vite",
     "build": "tsc -b && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "prettier:write": "prettier . --write",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "test": "jest --config jest.config.mjs"
```

Comment:
why not vite, since you're already using it to build the project? 😁

Reply:
ah very true, just used to it.  Will change

Reply:
fixed